pandas/api/tests/test_api.py

```diff
@@ -229,6 +229,6 @@ def test_deprecation_access_obj(self):
             pd.datetools.monthEnd
 
 if __name__ == '__main__':
-    import nose
-    nose.runmodule(argv=[__file__, '-vvs', '-x', '--pdb', '--pdb-failure'],
-                   exit=False)
+    import pytest
+
+    pytest.main([__file__, '-vvs', '-x', '--pdb'])
```

Comment:
Is it actually needed to keep those `if __name__ == '__main__':` parts? Does anybody use this? 
(but can of course remove this in another PR if we want this)